### PR TITLE
Plushie vendor, Eyepatch HUDs, and more fixes

### PR DESCRIPTION
**Pull request by ThePainkiller**
## Map changes:

Adds a wall toy vendor on the Lonestar cargo lobby that sells all plushies
image

Erases all the athletic lockers from the bathrooms, replaces one of the suit storage units with the underwear wardrobe, moves the bathrobes to the racks along with the towels that were already there
image

## Everything else new added:

- Adds a toy vending machine holding almost every single plushie and toy, sprite by yours truly
- Adds DND Figurines to the Game Emporium so that people can really play DND with actual characters
- Fixes duplicate name on olive suit skirt that lead to loadout issues
- Resprites a couple toy figures to better fit thematically (colonist, CBO, doctor, marshal officer, supply spec and WO)
- Server apropiate carp plushie sprite
- Medical injector comes with dexalin plus and tramadol
- Blackshield gloves on loadout only for Blackshield jobs
- Minor typo fixes
- Dormitories APC no longer starts turned off
- Fixes the Von-Krabin stimulator not being able to be sacrificed to the Artificer Perfection Cube
- Eyepatch HUDs available on Soteria, Security and Guild loadout, fully functional and can fit either eye!
image
- Soteria Lifeline Techs get a gear selection briefcase on their hardcase now, for now it only contains the "Bullpip" SMG but leaves a path open if other options are to be added.
- Adds six varieties of Hakamas and a Haori to modular suit selection and suit jacket selection
- New katana sprite courtesy of Kavrick, in hands edited to reflect change

<hr>

## Changelog
:cl:
add: Adds a new vendor to Lonestar Cargo desk that sells all manner of toys and plushies
add: Adds six varieties of Hakamas to modular suit selection, as well as a Haori to suit jacket selection. Available on the formal suits vendor too.
add: Blackshield gloves to security loadout
add: Adds a katana-specific sheath, found on holster loot piles or taken from loadout.
add: Lifeline Technicians get a case inside their hardcase to select their primary weapon. Currently only the Bullpip SMG is available, but the item is there in case any other sensible guns are planned to be added.
add: Eyepatch HUDs available on Soteria, Security and Guild loadout, fully functional and can fit either eye
tweak: Adds DND Figurines to the Game Emporium so that people can really play DND with actual characters
tweak: Appropiate carp plushie sprite
fix: Fixes duplicate name on olive suit skirt that lead to loadout issues
fix: Dormitories APC no longer starts turned off
fix: Fixes the Von-Krabin stimulator not being able to be sacrificed to the Artificer Perfection Cube
balance: Medical chem injector has also dexplus and tramadol now
balance: Gear now properly has an increased firing cost as stated by description
imageadd: New Nadezhda-themed icons for toys (colonist, CBO, doctor, marshal officer, supply spec and WO)
imageadd: New sprites for the Katana courtesy of @Kavrick#5690 and katana-specific sheath sprite made by me.
spellcheck: Fixed a few typos
/:cl:

### CODE REVIEW

File: code/game/machinery/vendor/cloathing.dm [hunk 1]
```diff
@@ -806,3 +806,270 @@
 		/obj/item/clothing/suit/storage/suitjacket/shortcardigan_blueandbrown = 130
 	)
 
+/obj/machinery/vending/plushies
+	name = "Toys 4 You"
+	desc = "Selling toys of all kinds for your inner child."
```
Comment (Trilbyspaceclone) [inline code/game/machinery/vendor/cloathing.dm:811]: please have a different desc

File: code/game/machinery/vendor/cloathing.dm [hunk 2]
```diff
@@ -806,3 +806,270 @@
...
+	density = FALSE
+	light_color = COLOR_LIGHTING_GREEN_BRIGHT
+	icon_state = "amogus"
+	icon_deny = "amogus-deny"
+	product_slogans = "You're never too old to play with toys, of any kind!;As seen in posters among us toymakers!;Gotta collect 'em all!"
```
Comment (Trilbyspaceclone) [inline code/game/machinery/vendor/cloathing.dm:816]:
```suggestion
	product_slogans = "Buy one for everyone!;As seen in posters among us toymakers!;Gotta collect 'em all!"
```

File: code/modules/clothing/spacesuits/rig/modules/utility.dm [hunk 3]
```diff
@@ -268,16 +268,17 @@
...
 	max_injection_amount = 60
 	max_beakers = 6
 	usable = 0
 	selectable = 1
 	disruptive = 1
 	initial_beakers = list(
+		list(/obj/item/reagent_containers/glass/beaker/large, "dexalinp",60),
+		list(/obj/item/reagent_containers/glass/beaker/large, "tramadol",60),
 		list(/obj/item/reagent_containers/glass/beaker/large, "bicaridine", 60),
-		list(/obj/item/reagent_containers/glass/beaker/large, "inaprovaline",60),
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/spacesuits/rig/modules/utility.dm:280]: Please do not remove inaprovaline from this

Comment (ThePainkiller) [reply code/modules/clothing/spacesuits/rig/modules/utility.dm:280]: After livermed Inaprovaline has become, once more, completely useless as a chem to stabilize a patient, but sure.

**Comment by ThePainkiller** [comment]
> Looks good Seb! I would recommend ya' slide a few of the obvious antag stuff behind the "hacked/contraband" list for the vendors though. Excel being the one that other vendors try to hide.

You mean like the Excelsior and merc bobblehead? Sure, they would fit in as contraband, I'll move them there for the sake of having a reason to hack it

**Comment by ThePainkiller** [comment]
Hatred, anger even

File: code/modules/projectiles/guns/energy/cog.dm [hunk 4]
```diff
@@ -24,17 +24,75 @@
...
+	penetration_multiplier = 0.8 // +0.2 when unfolded = base Cog
+	projectile_type = /obj/item/projectile/beam/stun
+	w_class = ITEM_SIZE_NORMAL // Starts folded, bulk added when unfolded FAR surpasses BULKY class
+	folding_stock = TRUE // Foldable stock for easy carry
+	price_tag = 950
+	fire_delay = 10
+	matter = list(MATERIAL_STEEL = 10, MATERIAL_PLASTIC = 10, MATERIAL_SILVER = 10)
+	init_firemodes = list(
+		list(mode_name="stun", mode_desc="fires a concentrated stun beam", projectile_type=/obj/item/projectile/beam/stun, charge_cost = 50, icon="stun", fire_sound='sound/weapons/Taser3.ogg'),
+		list(mode_name="lethal", mode_desc="fires a concentrated laser blast", projectile_type=/obj/item/projectile/beam, charge_cost = 50, icon="kill", fire_sound='sound/weapons/Laser.ogg')
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/energy/cog.dm:56]:
```suggestion
		list(mode_name="lethal", mode_desc="fires a concentrated laser blast", projectile_type=/obj/item/projectile/beam/weak, charge_cost = 50, icon="kill", fire_sound='sound/weapons/Laser.ogg')
```

Comment (Trilbyspaceclone) [reply code/modules/projectiles/guns/energy/cog.dm:56]: It being smaller dosnt account for only 10% less damage when unfolded

**Comment by ThePainkiller** [comment]
My god at this point just fucking scrap the gun, I'm done. I replaced the lethal laser with the bolt and kept the normal disabling laser and now it's just a martin with a Zeus stunbolt, far from the concept I originally meant for the gun. I'm fucking done with this.

**Pull request merged**